src/yadmark/benchmark/runner.py

```diff
@@ -53,7 +53,8 @@ def run_me(self, theory, ocard, pdf):
             method = runcards.Legacy.MOD_EV2METHOD.get(theory["ModEv"], theory["ModEv"])
             method = dictlike.load_enum(types.EvolutionMethod, method)
             method = couplings_mod_ev(method)
-            masses = [mq.value**2 for mq in new_eko_theory.quark_masses]
+            masses = [np.inf, np.inf, np.inf]
```

Comment:
?

If you want to use FFNS3, you should change the `thresholds_ratios`, not the masses.